### ISSUE TITLE
Android keyboard disappear just after a click on the global search input

### ISSUE THREAD

**Issue by soullivaneuh** [sonata-project/SonataAdminBundle#5045]
#### Sonata packages

```
$ composer show --latest 'sonata-project/*'
sonata-project/admin-bundle              3.32.0 3.33.0 The missing Symfony Admin Generator
sonata-project/block-bundle              3.12.0 3.12.1 Symfony SonataBlockBundle
sonata-project/cache                     2.0.1  2.0.1  Cache library
sonata-project/core-bundle               3.9.1  3.9.1  Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.3.1  2.3.1  Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.0.2  1.0.2  Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.4.2  3.4.2  Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/easy-extends-bundle       2.5.0  2.5.0  Symfony SonataEasyExtendsBundle
sonata-project/exporter                  1.8.0  1.8.0  Lightweight Exporter library
sonata-project/intl-bundle               2.4.0  2.4.0  Symfony SonataIntlBundle
```

#### Symfony packages

```
$ composer show --latest 'symfony/*'
symfony/asset                v4.0.6  v4.0.6  Symfony Asset Component
symfony/browser-kit          v4.0.6  v4.0.6  Symfony BrowserKit Component
symfony/cache                v4.0.6  v4.0.6  Symfony Cache component with PSR-6, PSR-16, and tags
symfony/class-loader         v3.4.6  v3.4.6  Symfony ClassLoader Component
symfony/config               v3.4.6  v4.0.6  Symfony Config Component
symfony/console              v3.4.6  v4.0.6  Symfony Console Component
symfony/css-selector         v4.0.6  v4.0.6  Symfony CssSelector Component
symfony/debug                v4.0.6  v4.0.6  Symfony Debug Component
symfony/debug-bundle         v4.0.6  v4.0.6  Symfony DebugBundle
symfony/dependency-injection v3.4.6  v4.0.6  Symfony DependencyInjection Component
symfony/doctrine-bridge      v3.4.6  v4.0.6  Symfony Doctrine Bridge
symfony/dom-crawler          v4.0.6  v4.0.6  Symfony DomCrawler Component
symfony/dotenv               v4.0.6  v4.0.6  Registers environment variables from a .env file
symfony/event-dispatcher     v3.4.6  v4.0.6  Symfony EventDispatcher Component
symfony/expression-language  v3.4.6  v4.0.6  Symfony ExpressionLanguage Component
symfony/filesystem           v3.4.6  v4.0.6  Symfony Filesystem Component
symfony/finder               v3.4.6  v4.0.6  Symfony Finder Component
symfony/flex                 v1.0.71 v1.0.78
symfony/form                 v3.4.6  v4.0.6  Symfony Form Component
symfony/framework-bundle     v3.4.6  v4.0.6  Symfony FrameworkBundle
symfony/http-foundation      v3.4.6  v4.0.6  Symfony HttpFoundation Component
symfony/http-kernel          v3.4.6  v4.0.6  Symfony HttpKernel Component
symfony/inflector            v4.0.6  v4.0.6  Symfony Inflector Component
symfony/intl                 v4.0.6  v4.0.6  A PHP replacement layer for the C intl extension that includes additional data from the ICU library.
symfony/monolog-bridge       v4.0.6  v4.0.6  Symfony Monolog Bridge
symfony/monolog-bundle       v3.2.0  v3.2.0  Symfony MonologBundle
symfony/options-resolver     v3.4.6  v4.0.6  Symfony OptionsResolver Component
symfony/phpunit-bridge       v3.4.6  v4.0.6  Symfony PHPUnit Bridge
symfony/polyfill-intl-icu    v1.7.0  v1.7.0  Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-mbstring    v1.7.0  v1.7.0  Symfony polyfill for the Mbstring extension
symfony/polyfill-php56       v1.7.0  v1.7.0  Symfony polyfill backporting some PHP 5.6+ features to lower PHP versions
symfony/polyfill-php70       v1.7.0  v1.7.0  Symfony polyfill backporting some PHP 7.0+ features to lower PHP versions
symfony/polyfill-php72       v1.7.0  v1.7.0  Symfony polyfill backporting some PHP 7.2+ features to lower PHP versions
symfony/polyfill-util        v1.7.0  v1.7.0  Symfony utilities for portability of PHP codes
symfony/process              v3.4.6  v4.0.6  Symfony Process Component
symfony/property-access      v3.4.6  v4.0.6  Symfony PropertyAccess Component
symfony/property-info        v3.4.6  v4.0.6  Symfony Property Info Component
symfony/routing              v3.4.6  v4.0.6  Symfony Routing Component
symfony/security             v3.4.6  v4.0.6  Symfony Security Component
symfony/security-acl         v3.0.1  v3.0.1  Symfony Security Component - ACL (Access Control List)
symfony/security-bundle      v3.4.6  v4.0.6  Symfony SecurityBundle
symfony/serializer           v3.4.6  v4.0.6  Symfony Serializer Component
symfony/stopwatch            v4.0.6  v4.0.6  Symfony Stopwatch Component
symfony/swiftmailer-bundle   v3.2.0  v3.2.1  Symfony SwiftmailerBundle
symfony/templating           v4.0.6  v4.0.6  Symfony Templating Component
symfony/translation          v3.4.6  v4.0.6  Symfony Translation Component
symfony/twig-bridge          v3.4.6  v4.0.6  Symfony Twig Bridge
symfony/twig-bundle          v3.4.6  v4.0.6  Symfony TwigBundle
symfony/validator            v3.4.6  v4.0.6  Symfony Validator Component
symfony/var-dumper           v4.0.6  v4.0.6  Symfony mechanism for exploring and dumping PHP variables
symfony/web-profiler-bundle  v4.0.6  v4.0.6  Symfony WebProfilerBundle
symfony/yaml                 v3.4.6  v4.0.6  Symfony Yaml Component
```

#### PHP version

```
$ php -v
PHP 7.1.15 (cli) (built: Mar 14 2018 15:00:01) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
    with Xdebug v2.6.0, Copyright (c) 2002-2018, by Derick Rethans
```

## Subject

We currently can't make a global search on Android. Works on IOS.

## Steps to reproduce

On an Android device, go on a Sonata page and try to tap on the global search input.

## Expected results

The keyboard appear and I can do my search.

## Actual results

The keyboard did appeared but disappeared just after that.

Funny note: If I tap a lot of time on the input, I will finally have my keyboard stick on the screen.

**Comment by jmros** [comment]
same problem here

**Comment by GWillmann** [comment]
It seems to be solved since release 2.4.0 of AdminLTE.

See the PR that fixed the issue here: 
[https://github.com/almasaeed2010/AdminLTE/pull/1416](https://github.com/almasaeed2010/AdminLTE/pull/1416)

See the release that packaged the fix here: [https://github.com/almasaeed2010/AdminLTE/releases/tag/v2.4.0](https://github.com/almasaeed2010/AdminLTE/releases/tag/v2.4.0)

SonataAdminBundle is currently using version 2.3.8 of AdminLTE. 

Could we consider upgrading to 2.4.0 or greater (latest release is 2.4.5) to help fix this issue?

Have a nice day!

**Comment by jmros** [comment]
yes. still relevant.
we need upgrade the adminlte scripts. but i don't know if is posible or not.

**Comment by jmros** [comment]
this isue still relevant

**Comment by VincentLanglet** [comment]
> It seems to be solved since release 2.4.0 of AdminLTE.
> 
> See the PR that fixed the issue here:
> [ColorlibHQ/AdminLTE#1416](https://github.com/ColorlibHQ/AdminLTE/pull/1416)
> 
> See the release that packaged the fix here: https://github.com/almasaeed2010/AdminLTE/releases/tag/v2.4.0
> 
> SonataAdminBundle is currently using version 2.3.8 of AdminLTE.
> 
> Could we consider upgrading to 2.4.0 or greater (latest release is 2.4.5) to help fix this issue?
> 
> Have a nice day!

Sonata 4 use `"admin-lte": "^2.4",` so I assume it's fixed on master.
